### PR TITLE
set_embed_color, embed_color and get_message added

### PR DESCRIPTION
**Pull request by NightSlasher35**
## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/client.py [hunk 1]
```diff
@@ -241,6 +241,58 @@ def __init__(
...
+        --------
+        Optional[:class:`.Message`]
+            The message asked for."""
+        return utils.get(self.cached_messages, id=id)
+
+    @property
+    def embed_color(self):
+        """Optional[:class:`.Colour`]: The default embed colour that is
+        being used for all embed if no colour is passed."""
+        col = os.getenv("DEFAULT_EMBED_COLOR")
```
Comment (Gnome-py) [inline discord/client.py:261]: No, never use env vars like this, instead store this under `discord.Embed.default_colour`

File: discord/client.py [hunk 2]
```diff
@@ -241,6 +241,58 @@ def __init__(
...
+            Pass either an instance of discord.Color or a valid
+            0x****** HEX value.
+        Returns
+        --------
+        :class:`.Colour`
+            The new color that has been set as default embed color.
+        """
+        if isinstance(color, (Color, Colour)):
+            os.environ['DEFAULT_EMBED_COLOR'] = str(hex(color))
+            return color
```
Comment (Gnome-py) [inline discord/client.py:283]: Same here, do not use env vars

File: discord/client.py [hunk 3]
```diff
@@ -241,6 +241,58 @@ def __init__(
...
+        else:
+            try:
+                HEX = re.compile(r'^(#)[A-Fa-f0-9]{6}$')
```
Comment (Gnome-py) [inline discord/client.py:286]: Why are you compiling this regex per invoke, store global.

**Comment by WhoTheOOF** [comment]
Going to go with Gnome here.

**Comment by GnomedDev** [comment]
This has gone stale, `get_message` is a kinda good idea though.

**Pull request closed without merging**